### DISCUSSION TITLE
"Titanic galaxy cluster collision in the early universe challenges standard cosmology"

### DISCUSSION THREAD

**Discussion by RedshiftDrift** [a-cosmology-group/acg#145] [Cosmology Models]
[https://phys.org/news/2023-09-titanic-galaxy-cluster-collision-early.html](https://phys.org/news/2023-09-titanic-galaxy-cluster-collision-early.html)
by Elena Asencio, Indranil Banik, and Pavel Kroupa, of course.

The [open access paper](https://iopscience.iop.org/article/10.3847/1538-4357/ace62a).

**Comment by RedshiftDrift** [comment]
["Three critical tests alone in this year 2024 find dark matter and the LCDM model to be incorrect"](https://darkmattercrisis.wordpress.com/2024/07/08/93-three-critical-tests-alone-in-this-year-2024-find-dark-matter-and-the-lcdm-model-to-be-incorrect/)
by Pavel Kroupa and Eda Gjergo

**Reply by mikehelland** [reply]
Minor tweak to the URL (an errant [):

https://darkmattercrisis.wordpress.com/2024/07/08/93-three-critical-tests-alone-in-this-year-2024-find-dark-matter-and-the-lcdm-model-to-be-incorrect/

**Reply by RedshiftDrift** [reply]
Oops! ...corrected.  Thanks.

**Reply by LifeIsStrange** [reply]
Thanks for sharing!! I somehow missed the first and third paper of the blog despite being groundbreaking results.
I did not know about Chandrasekhar dynamical friction.
About it, they mention GAIA astrometry, I wonder if their assumptions are sensitive to mass estimate because our estimates of the Milky Way (and I presume our satellites) are largely erroneous because of complexities in the Jeans equation (the star halo is not spherical and is tilted) and because GAIA accuracy non linearly drift with distance.

The third observation though, is extremely interesting, the mass uncertainty might be underevaluated but if I understand correctly it is simpler to estimate indirectly the masses of distant galaxies (and star clusters?) than it is for the Milky Way.

**Reply by mikehelland** [reply]
>if I understand correctly it is simpler to estimate indirectly the masses of distant galaxies (and star clusters?) than it is for the Milky Way.

Why's that?

**Reply by LifeIsStrange** [reply]
I suppose it is because we are within the Milky Way which obscure more our understanding of its shape and luminosity than if we were outside of it.
As for the magellanic clouds the situation is different but their history is complex and they have it seems suffered from many collisions which makes their shape complex.
On the contrary I suppose the uncertainty of the mass of Andromeda must be smallish even though specifically for some atoms/molecules there's a lot we ignore, for example, strikingly, can you imagine that we only extremely recently discovered (in 2023) a galactic sized blue arc neighboring Andromeda? (literally the closest non dwarf galaxy)
It was discovered by amateurs IIRC because they were the first to look at Andromeda in the oxygen III band, which is not an original idea at all but they were the first to do long exposure to see it.
It is an immense example of the hidden matter we are imminently about to discover with the ongoing IFU panchromatic spectroscopic surveys.
Such a pretty picture and gigantic structure, hiding in plain sight
https://www.astrobin.com/full/1d8ivk/0/

Here is a counterpoint though that while there might be some extended dust and some gases halos hidden, the main prospect of discovering hydrogen halos currently seems to indicate such hydrogen halos are not widespread

https://tritonstation.com/2024/04/26/the-mhongoose-survey-of-atomic-gas-in-and-around-galaxies/

**Reply by mikehelland** [reply]
>I suppose it is because we are within the Milky Way which obscure more our understanding of its shape and luminosity than if we were outside of it.

Being within the Milky Way gives us the ability to make measurements through parallax, and GAIA operates over a larger range of time than any other extrapolation of stars in external galaxies. 

It seems to me the most direct, empirical measurements are discarded  because of ... not agreeing with dark matter or MOND. 

Maybe they're both garbage?

**Reply by LifeIsStrange** [reply]
Gaia measurements are only robust at mid distance. Moreover even if we were to assume they were robust enough, this data alone is useless if you don't model properly the shape and radial velocities of the Milky Way halo (Cf H3 study I sent you before)
We should get a clearer picture with ongoing spectroscopic surveys.
While Gaia data is hard to use to assess the galactic rotation of the Milky Way and its mass, it can still derive robust conclusions via other tests, e.g. Wide binaries accelerations, on that topic the discrepancy between banik result and Chae are completely dissonant and the research is a mess, I don't know what to conclude from it but it's likely GAIA DR4 will give us a definite answer.

**Reply by mikehelland** [reply]
Mid distance?

What's the motion of stars in other galaxies based on?

.

**Reply by LifeIsStrange** [reply]
I serendipitously stumbled upon this paragraph

> Gaia DR3 data, after accounting for statistical corrections, indicates a distance of 4.1+0.3 −0.2 kpc.

> If, however, the remnant’s distance was somehow actually ∼ 4 kpc like that of CI Cam, then its angular diameter of 2.5◦ ×3.0◦ would imply unlikely dimensions of 180 pc × 215 pc ranking it as perhaps the largest and oldest Galactic SNR known despite its bright [O III] filaments indicating relatively high shock velocities.

https://www.prophy.science/article/178403947/
Which is an example that GAIA has high uncertainty past (and even below IIRC) 1 kiloparsec.

The motion of stars in distant (not local group) galaxies are in most cases, if I understand correctly, not resolved (unless exceptional gravitational lens maybe of star cluster) 
The mass, metallicity and rotation curve of the galaxy are inferred from the total galactic emission which supposedly work despite many stars being effectively dust obscured.

**Reply by mikehelland** [reply]
Just to clarify.... Are you saying we know the movement of stars in other galaxies better than we know them in the Milky Way?

**Reply by LifeIsStrange** [reply]
I believe we have no idea of the motion of individual stars in distant galaxies beyond the local group.
I mean except maybe for some giant stars like supernovaes, of which maybe e.g. via asteroseismology and spectroscopy some inferences, about their rotation, radial velocities and maybe astrometry might be possible, with low accuracy.
